pyas2/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.urls import path
 from django.contrib.auth.decorators import login_required
+from django.urls import include, path
```

Comment:
why these changes?

Reply:
That was committed by mistake. I removed it.